pkg/sql/show.go

```diff
@@ -56,6 +56,10 @@ var varNames = func() []string {
 	sort.Strings(res)
 	return res
 }()
+var varRequireExplicitTxn = map[string]struct{}{
```

Comment:
Make `varGen` above a `map[string]struct{ bool, func }` to embed this information there.